client/python/ovmsclient/lib/README.md

```diff
@@ -15,7 +15,7 @@ cd model_server/client/python/ovmsclient/lib
 
 ## Installation
 
-**Note:** The client library requires Python in version >= 3.6.
+**Note:** The client library requires Python in version >= 3.7.
```

Comment:
How is that? Why 3.6 is no longer valid?

Reply:
@dtrawins